index.js

```diff
@@ -1,9 +1,12 @@
-const nlp = require('compromise');
-const nlpPronounce = require('compromise-pronounce');
-nlp.extend(nlpPronounce);
+import nlp from 'compromise';
+import speechPlugin from 'compromise-speech'
+nlp.plugin(speechPlugin)
 
-function pronounce(text) {
-  return nlp(text).pronounce().map(o => o.pronounce).toString();
+function pronounce(pronounceword) {
+  let doc = nlp(pronounceword);
+  let spechword = decodeURI(doc.terms().soundsLike());
```

Comment:
There is a typo in the variable declaration. (Please follow CamelCase here too)
Same in return below.